AUTHORS.md

```diff
@@ -0,0 +1,14 @@
+<!--
+ - SPDX-FileCopyrightText: 2020-2024 Nextcloud GmbH and Nextcloud contributors
+ - SPDX-License-Identifier: GPL-3.0-or-later
+-->
+# Authors
+
+- Andy Scherzinger <info@andy-scherzinger.de>
+- Christoph Wurst <christoph@winzerhof-wurst.at>
+- Ferdinand Thiessen <opensource@fthiessen.de>
+- Grigorii K. Shartsev <me@shgk.me>
+- John Molakvoæ <skjnldsv@protonmail.com>
+- Julius Härtl <jus@bitgrid.net>
+- Pytal <24800714+Pytal@users.noreply.github.com>
+- Roeland Jago Douma <rullzer@users.noreply.github.com>
```

Comment:
Is it the list of contributors, or the list of contributors being listed in the old @copyright comments before migrating to SPDX header?

Reply:
Did you run

https://github.com/AndyScherzinger/machine-setup/blob/main/SETUP.md#useful-git-commands

For gaining authors from git as well?